src/lib/appManagers/appDraftsManager.ts

```diff
@@ -30,12 +30,8 @@ export class AppDraftsManager {
   private getAllDraftPromise: Promise<void> = null;
 
   constructor() {
-    appStateManager.getState().then(state => {
-      this.drafts = state.drafts;
-
-      appStateManager.addEventListener('save', async() => {
-        appStateManager.pushToState('drafts', this.drafts);
-      });
+    sessionStorage.get('drafts').then(drafts => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
    sessionStorage.get("drafts").then(drafts => {
```